update-eks-kubeconfig/update-eks-kubeconfig.sh

```diff
@@ -3,40 +3,58 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-CLUSTERS=(
+CLUSTERS_PROD=(
   eks-pac-test-eu
   eks-pac-test-us
 
 )
+CLUSTERS_DEV=(
```

Comment:
The account is called "test" - `ft-tech-content-platform-test (070529446553)`. So the name `TEST_ACCOUNT_CLUSTERS` seems better to me. The key work `test` should be used to refer to the account.

Reply:
Renamed as requested